extension/src/popup/views/Account/index.tsx

```diff
@@ -183,25 +180,7 @@ export const Account = () => {
             setLoading={setLoading}
           />
           {/* <BlockaidAnnouncement /> */}
-          <View.Content
```

Comment:
this looks like it was a result of a bad merge. This button now appears in the `...` dropdown on Account

Reply:
ah yup that was my mistake, thanks!